docs/getting_started/index.rst

```diff
@@ -144,42 +179,50 @@ try running some command line tools:
 .. code-block:: console
 
     $ ctapipe-info --all
-    $ ctapipe-camdemo --camera=NectarCam  # try --help for more info
+    $ ctapipe-process -i dataset://gamma_prod5.simtel.zst -o test.h5  # try --help for more info
 
 To update to the latest development version (merging in remote changes
 to your local working copy):
 
 .. code-block:: console
 
-   $ git pull upstream master
+   $ git fetch upstream
+   $ git merge upstream/main # or rebase, see above
 
 ---------------------------------------
 Developing a new feature or code change
 ---------------------------------------
 
-We adhere to the PEP8 coding style (see our :doc:`/development/style-guide`).
-To enforce this, setup the
+We are using the ``black`` and ``isort`` auto-formatters for automatic 
+adherence to the code style (see our :doc:`/development/style-guide`).
+To enforce running these tools whenever you make a commit, setup the
 `pre-commit hook <https://pre-commit.com/>`_::
 
     $ pre-commit install
 
-You should always create a branch when developing some new code (unless it is
-a very small change).  Generally make a new branch for each new feature, so
-that you can make pull-requests for each one separately and not mix code
-from each.  Remember that ``git switch <name>`` switches between branches,
+You should always create a new branch when developing some new code.
+Make a new branch for each new feature, so that you can make pull-requests
+for each one separately and not mix code from each.
+Remember that ``git switch <name>`` switches between branches,
 ``git switch -c <name>`` creates a new branch, and ``git branch`` on it's own
 will tell you which branches are available and which one you are currently on.
```

Comment:
Should we note that a newish git is required for that and link to `checkout` just in case?

Reply:
Could add a [footnote](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes)